game/src/edit/roads.rs

```diff
@@ -0,0 +1,80 @@
+use map_gui::tools::PromptInput;
+use map_model::{Direction, LaneType, Road};
+use widgetry::{EventCtx, State};
+
+use crate::app::{App, Transition};
+use crate::edit::apply_map_edits;
+
+/// Specify the lane types for a road using a text box. This is a temporary UI to start
+/// experimenting with widening roads. It'll be replaced by a real UI once the design is ready.
+pub fn prompt_for_lanes(ctx: &mut EventCtx, road: &Road) -> Box<dyn State<App>> {
+    let r = road.id;
+    PromptInput::new(
+        ctx,
+        "Define lanes_ltr",
+        lanes_to_string(road),
+        Box::new(move |string, ctx, app| {
+            // We're selecting a lane before this, but the ID is probably about to be invalidated.
+            app.primary.current_selection = None;
+
+            let mut edits = app.primary.map.get_edits().clone();
+            edits.commands.push(app.primary.map.edit_road_cmd(r, |new| {
+                new.lanes_ltr = string_to_lanes(string.clone());
+            }));
+            apply_map_edits(ctx, app, edits);
+            Transition::Multi(vec![Transition::Pop, Transition::Pop])
+        }),
+    )
+}
+
+fn lanes_to_string(road: &Road) -> String {
+    // TODO Assuming driving on the right.
```

Comment:
In other words, this doesn't currently work for UK et. al?

Reply:
Correct and no intention to fix, because this string-based lane editor is super temporary